### ISSUE TITLE
CTAP proxy: add SSH (ecdsa-sk and ed25519-sk) support

### ISSUE THREAD

**Issue by zpc0** [QubesOS/qubes-issues#9002]
### The problem you're addressing (if any)
OpenSSH supports auth via FIDO2 device, but Qubes CTAP proxy doesn't support this.
Currently I attach FIDO2 USB device to coding qube manually. :cry:

This will depends on #9001, because OpenSSH uses libfido2.

### The solution you'd like
Add OpenSSH support to CTAP proxy.

### The value to a user, and who that user might be
Qubes users gets strong SSH auth without `qubes-usb-proxy`.

**Comment by zpc0** [comment]
duplicate of #6480...
